apis/Google.Bigquery.V2/Google.Bigquery.V2.Snippets/BigqueryClientSnippets.cs

```diff
@@ -647,6 +648,600 @@ public void ParameterizedQuery_PositionalParameters()
             Assert.DoesNotContain(new { Name = "Nadia", Score = 1320L, Level = 2L }, resultsList);
         }
 
+        [Fact]
+        public async Task QueryOverviewAsync()
+        {
+            string projectId = _fixture.ProjectId;
+
+            // Sample: QueryOverview
+            var client = await BigqueryClient.CreateAsync(projectId);
```

Comment:
Ack. I think these were some of our first samples, before we'd worked that out. Will have a look through.